### DISCUSSION TITLE
SSL Certificate Renewal Perplexity

### DISCUSSION THREAD

**Discussion by bkrepp-ctps** [requarks/wiki#6406] [General]
I just renewed the SSL certificate for one of my organization's Wikis, but noticed something that was perplexing.
I renewed the certificate by pressing the 'Renew Certificate' button on the Admin -> SSL page.
After the renewal completed, the Admin -> SSL page listed the certificate expiration date as **5/11/2023** - not what I'd expect.
When I inspected the certificate for this wiki in the Google Chrome browser by clicking the "lock" icon in the address bar, selecting "Certificate is Secure" and then "Certificate is valid" the expiration date was given as **August 2, 2023** - what I would expect.

I'd like to think that the August 2, 2023 is the actual expiration date, and so wonder why WIkijs is displaying the expiration date as May 11.

Any ideas why this is happening?

Thanks and all best,
Ben Krepp

**Comment by NGPixel** [comment]
This is an known bug. Ignore the date reported by Wiki.js.
